packages/@ourworldindata/types/src/dbTypes/Images.ts

```diff
@@ -6,8 +6,8 @@ export interface DbInsertImage {
     defaultAlt: string
     filename: string
     id?: number
-    originalWidth?: number | null
-    originalHeight?: number | null
+    originalWidth: number
+    originalHeight: number
```

Comment:
These are the de-facto types in the DB, and making them non-nullable also doesn't cause any type issues.
So... I guess it's fine then?